src/main/java/com/connexta/store/adaptors/S3StorageAdaptor.java

```diff
@@ -84,32 +84,30 @@ public void store(
    */
   @Override
   @NotNull
-  public RetrieveResponse retrieve(@NotBlank final String key) throws RetrieveException {
-    log.info("Retrieving product in bucket \"{}\" with key \"{}\"", bucket, key);
+  public RetrieveResponse retrieve(@NotBlank final String key) throws RetrieveFileException {
+    log.info("Retrieving item in bucket \"{}\" with key \"{}\"", bucket, key);
```

Comment:
✏️ `file`?

Reply:
According to Architecture Team's definitions, `metadata` is not a `file`, and this class is now used for both `file`s and IRM.

Reply:
I just realized that I should update the name of the exception.

Reply:
Sorry, I got confused with another PR that I'm working on. This is only used for `file`s right now. I'll update it to say `file`.

Reply:
The only downside to using `file` to refer to the resource is that technically metadata is a file too.